global/test/TestTimer.hpp

```diff
@@ -60,11 +62,15 @@ class TestTimer : public CxxTest::TestSuite
         double elapsed_time = Timer::GetElapsedTime();
         TS_ASSERT_LESS_THAN_EQUALS(0, elapsed_time);
 
-        // Small amount of work in a loop that can't be optimized by unrolling
-        for (unsigned i=2; i<1000; )
+        // Small amount of work in a loop that can't be optimized by unrolling.
+        // Updated 2023-06-20 to make it even less likely the compiler will optimize this work away.
+        volatile double number = 0.0;
+        for (unsigned i = 1; i < 10'000; ++i)
         {
-            i += (int) floor(sqrt(i));
+            number += std::asinh(0.001 * i);
         }
+        TS_ASSERT_LESS_THAN(20'930.0, number);
```

Comment:
typo?

Reply:
If you're referring to the `20'930.0`, then no, this is legal since C++14:
https://en.cppreference.com/w/cpp/language/floating_literal

The compiler completely ignores it, but you can use it like you would use a comma to aid readability for long numbers.